src/app/pages/StoryPage/index.test.jsx

```diff
@@ -38,17 +41,18 @@ jest.mock('#containers/ChartbeatAnalytics', () => {
   return ChartbeatAnalytics;
 });
 
-const Page = ({ pageData, service }) => (
+const Page = ({ pageData, service, variant }) => (
   <StaticRouter>
     <ToggleContext.Provider value={{ toggleState, toggleDispatch: jest.fn() }}>
-      <ServiceContextProvider service={service}>
+      <ServiceContextProvider service={service} variant={variant}>
         <RequestContextProvider
           bbcOrigin="https://www.test.bbc.co.uk"
           isAmp={false}
           pageType={pageData.metadata.type}
           pathname={pageData.metadata.locators.assetUri}
           service={service}
           statusCode={200}
+          variant={pageData.variant}
```

Comment:
should this be `pageData.variant` when a `variant` prop is passed in?

Reply:
yes this is not needed 👍